tests/lean/conv_for_find.lean

```diff
@@ -29,6 +29,7 @@ example {a b : ℕ} : ∃ x, x * a = a * x :=  begin
   },
 end
 
+-- but this unfortunately doesn't work for `find` and `for`
```

Comment:
Once CI fails I'll update the error comparison file with the errors these tests produce.